src/Razor/test/testapps/Directory.Build.props

```diff
@@ -11,6 +11,7 @@
 
     <RazorSdkCurrentVersionProps>$(SolutionRoot)src\Microsoft.NET.Sdk.Razor\build\netstandard2.0\Sdk.Razor.CurrentVersion.props</RazorSdkCurrentVersionProps>
     <RazorSdkCurrentVersionTargets>$(SolutionRoot)src\Microsoft.NET.Sdk.Razor\build\netstandard2.0\Sdk.Razor.CurrentVersion.targets</RazorSdkCurrentVersionTargets>
+    <UsingToolMicrosoftNetCompilers>true</UsingToolMicrosoftNetCompilers>
```

Comment:
@natemcmaster looks like this wasn't enough to get the testapps to use this. At some point will have to look more into this.

Reply:
This property is only supported by the Arcade SDK, and it's a shortcut for including a PackageReference to Microsoft.NET.Compilers.Toolset. In a vanilla csproj, this property won't have an effect on the version of the compiler in use.

I'd recommend updating the global.json file in this repo to use 3.0.100-preview6-012262 instead and see if that works.

Reply:
Trying now